### PR TITLE
[WIP] ha: Use the portblock RA to block/unblock the port to the postgresql vip

### PR DESCRIPTION
**Pull request by vuntz**
This is recommended to avoid clients getting a ICMP port unreachable
message, that might make them think it's not worth reconnecting.

### CODE REVIEW

File: chef/cookbooks/postgresql/recipes/ha.rb [hunk 1]
```diff
@@ -79,17 +107,19 @@
...
     score "inf"
-    resources [fs_primitive, vip_primitive, service_name]
+    resources primitives
     action :create
   end
 
   pacemaker_order "o-#{service_name}" do
     score "Mandatory"
-    ordering "#{fs_primitive} #{vip_primitive} #{service_name}"
+    ordering "#{primitives.join(" ")}"
```
Comment (aspiers) [inline chef/cookbooks/postgresql/recipes/ha.rb]: Pointless string interpolation here, instead do:

    ordering primitives.join(' ')

**Comment by aspiers** [comment]
+1 apart from the interpolation nitpick.

File: chef/cookbooks/postgresql/recipes/ha.rb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/postgresql/recipes/ha.rb]: Fixed, this was copied from rabbitmq; I fixed it there too in the other PR.

**Comment by mjura** [comment]
:+1: looks ok

**Comment by rsalevsky** [comment]
@vuntz, this repository got merged into [crowbar-openstack](https://github.com/crowbar/crowbar-openstack).
Please open a new pull request there. __Do NOT merge here anymore!__

**Pull request closed without merging**